turnserver/config/config.go

```diff
@@ -0,0 +1,99 @@
+package config
```

Comment:
if main was built as a corba app; could get rid of most of these functions.... define flags for the vars and then they can be set using the flag or env vars... this would make it easier to deploy are bare-metal should someone decide to use bare-metal instead of containers

Reply:
agreed, can be put in the backlog I don't see it as an immediate required change